pkgs/sdk/server-ai/README.md

```diff
@@ -22,7 +22,7 @@ The .NET build tools should automatically load the most appropriate build of the
 
 ## Getting started
 
-Refer to the [SDK documentation](https://docs.launchdarkly.com/sdk/server-side/dotnet#getting-started) for instructions on getting started with using the SDK.
+Refer to the [SDK documentation](TODO) for instructions on getting started with using the SDK.
```

Comment:
TODO: update when available

Reply:
The link will be `https://docs.launchdarkly.com/sdk/ai/dotnet` but this PR will probably end up merging before the docs one. So you could also omit until after the docs are published if you want (I believe that's what Ryan did for Node).